portal/trigger_states/empro_domains.py

```diff
@@ -136,6 +137,7 @@ def eval_triggers(self):
                 triggers['domain'][domain] = dt.triggers
 
         triggers['source'] = {
+            'authored': FHIR_datetime.as_fhir(self.cur_qnr.authored),
             'qnr_id': self.cur_qnr.id,
```

Comment:
@pbugni Will this eventually contain the qnr_id for the post-intervention QNR?

Reply:
yes, but not under `triggers['source']` - probably `triggers['resolution']` will have it's own set of `{authored, qnr_id, ...}`